packages/pf-codemods/README.md

```diff
@@ -494,6 +494,30 @@ Additionally, the internal `aria-labelledby` for ProgressStep will only be appli
 
 This rule will raise a warning, but will not make any code changes.
 
+### contextSelector-update-deprecatedPath [(#8840)](https://github.com/patternfly/patternfly-react/pull/8840)
+
+We've deprecated the current implementation of ContextSelector. In order to continue using this deprecated implementation, the import path must be updated to our deprecated package and specifiers must be aliased. However, we suggest updating to our composable Menu implementation.
+
+The following imports will be affected by this deprecation:
+
+- ContextSelector
+- ContextSelectorItem
+- ContextSelectorFooter
+
+#### Examples
+
+In:
+
+```jsx
+import { ContextSelector, ContextSelectorItem } from '@patternfly/react-core';
+```
+
+Out:
+
+```jsx
+import { ContextSelector as ContextSelectorDeprecated, ContextSelectorItem as ContextSelectorItemDeprecated } from '@patternfly/react-core/deprecated';
+```
```

Comment:
Can we also add to the in/out an example of one of the components being used?

In: `<ContextSelector />`
Out: `<ContextSelectorDeprecated />`